# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix-gcc14.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- thunderbird-136.0/media/ffvpx/libavcodec/av1dec.c.vanilla	2025-03-13 21:30:25.250463336 +0100
+++ thunderbird-136.0/media/ffvpx/libavcodec/av1dec.c	2025-03-13 21:31:07.346461438 +0100
@@ -904,7 +904,7 @@
         ff_cbs_fragment_reset(&s->current_obu);
     }
 
-    s->dovi.logctx = avctx;
+    s->dovi.logctx = (AVContext *)avctx;
     s->dovi.cfg.dv_profile = 10; // default for AV1
     sd = ff_get_coded_side_data(avctx, AV_PKT_DATA_DOVI_CONF);
     if (sd && sd->size >= sizeof(s->dovi.cfg))
--- thunderbird-136.0/media/ffvpx/libavcodec/libdav1d.cvanilla	2025-03-13 21:31:26.017006670 +0100
+++ thunderbird-136.0/media/ffvpx/libavcodec/libdav1d.c	2025-03-13 21:31:53.989107092 +0100
@@ -293,7 +293,7 @@
     c->delay = res > 1 ? res : 0;
 #endif
 
-    dav1d->dovi.logctx = c;
+    dav1d->dovi.logctx = (AVContext *)c;
     dav1d->dovi.cfg.dv_profile = 10; // default for AV1
     sd = ff_get_coded_side_data(c, AV_PKT_DATA_DOVI_CONF);
     if (sd && sd->size >= sizeof(dav1d->dovi.cfg))
